README.md

```diff
@@ -55,7 +55,7 @@ Your task is to design the foundation of a user interface capable of identifying
    - From the transactions filtered previously, order the planets based on the total number of transactions, from the highest to the lowest.
 
 3. **Cumulative transaction values:**
-   - From the ordered list of planets, provide real-time cumulative values for all transactions in both ICS ("Imperial Crown Standard") and GCS ("Galactic Credit Standard") for each planet individually.
+   - From the ordered list of planets, provide real-time cumulative values for all transactions in both ICS ("Imperial Crown Standard") and GCS ("Galactic Credit Standard") for each planet individually. Furthermore, please also add the total real-time cumulative values for all transactions of all planets in both ICS and GCS.
```

Comment:
I think we misunderstood each other. I thought we said we wanted the total number of transactions to help us review the data in the test. Adding the total real-time cumulative values for all transactions of all planets in both ICS and GCS adds more complexity and length to our test which is probably already long enough.

Reply:
Right, but I feel like just asking the total amount of transactions is prone to show the right amount of transactions, but the wrong ICS value, thus not really helping us understand if the calculation is good. But it would provide a quick way to test at least the transactions amounts.